src/gtimelog/main.py

```diff
@@ -1,1950 +1,49 @@
 """An application for keeping track of your time."""
-import time
-import sys
-
-
-DEBUG = '--debug' in sys.argv
-
-
-if DEBUG:
-    def mark_time(what=None, _prev=[0, 0]):
-        t = time.time()
-        if what:
-            print("{:.3f} ({:+.3f}) {}".format(t - _prev[1], t - _prev[0], what))
-        else:
-            print()
-            _prev[1] = t
-        _prev[0] = t
-else:
-    def mark_time(what=None):
-        pass
-
-
-mark_time()
-mark_time("in script")
```

Comment:
I had a question about that btw... I never see this kind of spy behavior. Useally, this is usefull when you code and so you can spy python bytescode to see leaks and other stuffs with specific tools embedded in your debug environement. I think it add code that is not usefull (i mean 99.9999999% of the time)

Reply:
I put it back as much as i could with the new arch

Reply:
Yes, this is a relic from when I was trying to optimize gtimelog's startup time.  If you run `gtimelog --debug`, you'll see how it works.

It's definitely not something for everyday use, but I take a look every now and then to avoid regressing the startup time.

It may be fine to move the def mark_time() into a gtimelog/debug.py and import it at the very top of main.py, with an `# isort: skip - must be first import` comment so it won't be moved.  The first mark_time() invocation also belongs inside debug.py